frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1419,13 +1422,19 @@ export const surveyLogic = kea<surveyLogicType>([
         defaultInterval: [
             (s) => [s.survey],
             (survey: Survey): IntervalType => {
-                const start = dayjs(survey.start_date || survey.created_at)
-                const end = survey.end_date ? dayjs(survey.end_date) : dayjs()
-                const diffInWeeks = end.diff(start, 'weeks')
+                const start = getSurveyStartDateForQuery(survey)
+                const end = getSurveyEndDateForQuery(survey)
+                const diffInDays = dayjs(end).diff(dayjs(start), 'days')
```

Comment:
do you still need to `dayjs(x)`? the values are a dayjs already

Reply:
we actually return them as strings, not `dayjs`, because we call `.format` in the end on the functions

Reply:
true that